### DISCUSSION TITLE
Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by manferlo81** [OpenHantek/OpenHantek6022#218] [Ideas]
It would be interesting if the user could customize some shortcuts in order to change the "time per division" or "volts per division" or enable/disable channel 1 or 2, it should be very easy to implement as well. It's a simple implementation which will bring a lot of improvements in the ux.

**Comment by Ho-Ro** [comment]
> ... it should be very easy to implement as well.

Your pull request would be very welcome.

**Reply by manferlo81** [reply]
I will have a look at it later, I'll setup my environment for cpp, I will be on windows as I don't have Linux in any of my PCs. Thanks @Ho-Ro. Great job with this app BTW huuuuge improvements over the original one.
